index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <div id="main">
+      <div id="title">
+    <h1> Dr. Norman Borlaug</h1>
+      </div>
+
+    <p>The man who saved a billion lives</p>
+<div id="img-div">
+    <img id="image" src="tribute-page-main-image.jpg" alt="">
+    <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+  </div>
+
+    
+<div id="tribute-info">
+    <h3>Here's a time line of Dr. Borlaug's life:</h3>
+
+    <ul>
+      <li> <b>1914 </b> - Born in Cresco, Iowa
```

Comment:
you should not use the <br> tag for best html practices instead you have to add this line break using css.